packages/firebase_in_app_messaging/ios/firebase_in_app_messaging.podspec

```
@@ -1,24 +1,39 @@
 #
 # To learn more about a Podspec see http://guides.cocoapods.org/syntax/podspec.html
 #
+firebase_sdk_version = '6.26.0'
+if defined?($FirebaseSDKVersion)
+  Pod::UI.puts "#{pubspec['name']}: Using user specified Firebase SDK version '#{$FirebaseSDKVersion}'"
+  firebase_sdk_version = $FirebaseSDKVersion
+else
+  firebase_core_script = File.join(File.expand_path('..', File.expand_path('..', File.dirname(__FILE__))), 'firebase_core/ios/firebase_sdk_version.rb')
+  if File.exist?(firebase_core_script)
+    require firebase_core_script
+    firebase_sdk_version = firebase_sdk_version!
+    Pod::UI.puts "#{pubspec['name']}: Using Firebase SDK version '#{firebase_sdk_version}' defined in 'firebase_core'"
+  end
+end
+
 Pod::Spec.new do |s|
-  s.name             = 'firebase_in_app_messaging'
-  s.version          = '0.0.1'
-  s.summary          = 'In-App Messaging Plugin for Firebase'
-  s.description      = <<-DESC
-Flutter plugin for Firebase In-App Messaging.
-                       DESC
-  s.homepage         = 'https://github.com/FirebaseExtended/flutterfire/tree/master/packages/firebase_in_app_messaging'
+  s.name             = pubspec['name']
+  s.version          = library_version
+  s.summary          = pubspec['description']
+  s.description      = pubspec['description']
+  s.homepage         = pubspec['homepage']
   s.license          = { :file => '../LICENSE' }
-  s.author           = { 'Flutter Team' => 'flutter-dev@googlegroups.com' }
+  s.author           = 'The Chromium Authors'
   s.source           = { :path => '.' }
   s.source_files = 'Classes/**/*'
   s.public_header_files = 'Classes/**/*.h'
   s.dependency 'Flutter'
-  s.dependency 'Firebase'
-  s.dependency 'Firebase/InAppMessagingDisplay'
+  s.dependency 'firebase_core'
+  s.dependency 'Firebase/CoreOnly', "~> #{firebase_sdk_version}"
+  s.dependency 'Firebase/InAppMessaging', "~> #{firebase_sdk_version}"
   s.static_framework = true
 
   s.ios.deployment_target = '8.0'
-end
 
+  s.pod_target_xcconfig = {
```

Noticed the Firebase plugin header variables are missing here, but this is on master also, so we could add a TODO for now to add them in later.

```suggestion
  s.pod_target_xcconfig = {
    # TODO(Salakar): Enable once plugin refactored to support.
    # 'GCC_PREPROCESSOR_DEFINITIONS' => "LIBRARY_VERSION=\\@\\\"#{library_version}\\\" LIBRARY_NAME=\\@\\\"flutter-fire-fiam\\\"",
```